### ISSUE TITLE
first-time-contributor plugin not found (Yarn v2 Berry PnP)

### ISSUE THREAD

**Issue by tunnckoCore** [intuit/auto#757]
**Describe the bug**

I have it installed.

```
❯ yarn auto shipit -d          
⚠  warning   Could not find plugin: first-time-contributor
```

When remove it from the plugins array everything is fine. 

```
"@auto-it/all-contributors": "^7.17.0",
"@auto-it/conventional-commits": "^7.17.0",
"@auto-it/first-time-contributor": "^7.17.0",
"@auto-it/npm": "^7.17.0",
"@auto-it/omit-commits": "^7.17.0",
"@auto-it/released": "^7.17.0",
"auto": "^7.17.0",
```

**To Reproduce**

Using latest Yarn Berry `yarn auto shipit -d`

***

Other than that... Auto seems to work fine in whatever env ;P

**Comment by hipstersmoothie** [comment]
@tunnckoCore is this still an issue? I haven't been able to get pnp working on my machine very easily to test this and would love a PR ❤️

**Comment by tunnckoCore** [comment]
I'm not sure. May try later this week if I can get to that.

**Comment by hipstersmoothie** [comment]
Thanks!

**Comment by adierkens** [comment]
:rocket: Issue was released in `v9.17.1` :rocket:
